lesson_03/homework_03.py

```diff
@@ -0,0 +1,182 @@
+"""Модуль для задач с использованием строк и математических вычислений."""

```

Comment:
Linter shows me this :
```
./bin/lint ./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:135:5: W503 line break before binary operator
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:136:5: W503 line break before binary operator
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:137:5: W503 line break before binary operator
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:138:5: W503 line break before binary operator
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:153:9: E126 continuation line over-indented for hanging indent
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:154:9: W503 line break before binary operator
./T_HOMETASK/Lesson_03/03_Ludmila_Chunihina.py:175:80: E501 line too long (96 > 79 characters)
```
For the next hometask tasks please pay attention to linter warning - you need to fix them before commit
For this current task - you have second attempt until 26-Oct-2024 - try to fix linter errors mentioned above